### PR TITLE
ci: Do not add ppa:alexlarsson/flatpak repo for ppc64le

### PR DESCRIPTION
**Pull request by nitkon**
Do not add ppa:alexlarsson/flatpak repo for ppc64le
arch as it fails to add key resulting in CI failure.

Fixes: #1533

Signed-off-by: Nitesh Konkar niteshkonkar@in.ibm.com

### CODE REVIEW

File: .ci/setup_env_ubuntu.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ chronic sudo -E apt install -y build-essential python pkg-config zlib1g-dev
 echo -e "Install CRI-O dependencies available for Ubuntu $VERSION_ID"
 chronic sudo -E apt install -y libdevmapper-dev btrfs-tools util-linux
 
-if [ "$VERSION_ID" == "16.04" ]; then
+if [ "$VERSION_ID" == "16.04" ] && [ "$(arch)" != "ppc64le" ]; then
 	echo "Install os-tree"
```
Comment (grahamwhaley) [inline .ci/setup_env_ubuntu.sh]: not your doing I think, but could you lift the echo outside the `if`, as I think `os-tree` will be installed by the next step, no matter what... thx!

Comment (nitkon) [reply .ci/setup_env_ubuntu.sh]: Done.

**Comment by chavafg** [comment]
@jodh-intel, `os-tree` is a cri-o dependency, which iirc was only available through that repo for Ubuntu 16.04 amd64 arch

We also install this on other distros, e.g.
https://github.com/kata-containers/tests/blob/43525f643b7a5e4736673ca669aabdb486e3b54d/.ci/setup_env_fedora.sh#L40

**Comment by nitkon** [comment]
ARM CI failing after multiple restarts. Not related to this PR?
```
22:14:45 Install Qemu
22:19:51 package github.com/qemu/qemu: no Go files in /home/jenkins/workspace/kata-containers-tests-ARM-18.04-PR/go/src/github.com/qemu/qemu
22:19:51 can't load package: package github.com/kata-containers/packaging: no Go files in /home/jenkins/workspace/kata-containers-tests-ARM-18.04-PR/go/src/github.com/kata-containers/packaging
22:19:51 ~/workspace/kata-containers-tests-ARM-18.04-PR/go/src/github.com/qemu/qemu ~/workspace/kata-containers-tests-ARM-18.04-PR/go/src/github.com/kata-containers/tests
22:20:23 Already on 'master'
22:20:23 Process leaked file descriptors. See https://jenkins.io/redirect/troubleshooting/process-leaked-file-descriptors for more information
22:20:23 Your branch is up to date with 'origin/master'.
22:20:23 error: The following untracked working tree files would be overwritten by checkout:
22:20:23 	slirp/COPYRIGHT
22:20:23 Please move or remove them before you switch branches.
22:20:23 Aborting
22:20:24 Build step 'Execute shell' marked build as failure
```

**Comment by chavafg** [comment]
yeah, it is unrelated.
@Pennyzct any idea of the arm CI error?

**Pull request merged**